modules/bitgo/test/v2/unit/internal/tssUtils/eddsa.ts

```diff
@@ -565,18 +668,52 @@ describe('TSS Utils:', async function () {
         {
           from: 'bitgo',
           to: 'user',
-          publicShare: bitgoKeyShare.yShares[1].y + bitgoKeyShare.yShares[1].v + bitgoKeyShare.yShares[1].chaincode,
+          publicShare: params.bitgoKeyShare.yShares[1].y + params.bitgoKeyShare.yShares[1].v + params.bitgoKeyShare.yShares[1].chaincode,
           privateShare: encryptedBitgoToUserMessage.toString(),
         },
         {
           from: 'bitgo',
           to: 'backup',
-          publicShare: bitgoKeyShare.yShares[2].y + bitgoKeyShare.yShares[2].v + bitgoKeyShare.yShares[2].chaincode,
+          publicShare: params.bitgoKeyShare.yShares[2].y + params.bitgoKeyShare.yShares[2].v + params.bitgoKeyShare.yShares[2].chaincode,
           privateShare: encryptedBitgoToBackupMessage.toString(),
         },
       ],
     };
 
+    const userKeyId = userGpgKeyActual.keyPacket.getFingerprint().padStart(40, '0');
+    const bitgoToUserPublicShare = Buffer.from(
+      await sodium.crypto_scalarmult_ed25519_base_noclamp(Buffer.from(params.bitgoKeyShare.yShares[1].u, 'hex')),
+    ).toString('hex') + params.bitgoKeyShare.yShares[1].chaincode;
+    const bitgoToBackupPublicShare = Buffer.from(
+      await sodium.crypto_scalarmult_ed25519_base_noclamp(Buffer.from(params.bitgoKeyShare.yShares[2].u, 'hex')),
+    ).toString('hex') + params.bitgoKeyShare.yShares[2].chaincode;
+
+    bitgoKeychain.walletHSMGPGPublicKeySigs = await createWalletSignatures(
+      params.bitgoGpgKey.privateKey,
+      params.userGpgKey.publicKey,
+      params.userGpgKey.publicKey,
+      [
+        { name: 'commonKeychain', value: bitgoCombined.pShare.y + bitgoCombined.pShare.chaincode },
+        { name: 'userKeyId', value: userKeyId },
+        { name: 'backupKeyId', value: userKeyId },
+        { name: 'bitgoToUserPublicShare', value: bitgoToUserPublicShare },
+        { name: 'bitgoToBackupPublicShare', value: bitgoToBackupPublicShare },
+      ]
+    );
+
+    return bitgoKeychain;
+  }
+
+  async function nockBitgoKeychain(params: {
```

Comment:
is 'nock' a typo for 'mock'?

Reply:
You gotta ask that the authors of the `nock` package :D
We are calling these helper functions `nockSomething` because they use the `nock` library to setup HTTP level mocks.